test_regress/t/t_instrumentation.vlt

```diff
@@ -0,0 +1,9 @@
+// DESCRIPTION: Verilator: Verilog Test module
+//
+// This file ONLY is placed into the Public Domain, for any use,
+// without warranty, 2025 by Wilson Snyder.
+// SPDX-License-Identifier: CC0-1.0
+
+`verilator_config
+
+instrument -model "instrument_var" -id 0 -target "top_module.a1.b1.out"
```

Comment:
Check multiple targets, with different IDs.

Is there a better word for model?  --callback?